common/src/main/java/com/wynntils/screens/container/widgets/ContainerEditButton.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.screens.container.widgets;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.core.components.Models;
+import com.wynntils.screens.base.widgets.WynntilsButton;
+import com.wynntils.utils.render.FontRenderer;
+import com.wynntils.utils.render.RenderUtils;
+import com.wynntils.utils.render.Texture;
+import java.util.List;
+import net.minecraft.ChatFormatting;
+import net.minecraft.network.chat.Component;
+import org.lwjgl.glfw.GLFW;
+
+public class ContainerEditButton extends WynntilsButton {
```

Comment:
```suggestion
public class ContainerEditNameButton extends WynntilsButton {
```

Do we edit anything else?